packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -5322,6 +5244,92 @@ components:
         - description
         - name
         - code
+    ClientVisibility:
```

this could be generic in case of future re-use
```suggestion
    Visibility:
```

---

Refactored to use `Visibility` in the authorization process, while I removed it here in the m2m API as discussed [on Slack](https://pagopaspa.slack.com/archives/C08J74Y0H96/p1751885937811279?thread_ts=1751640718.951539&cid=C08J74Y0H96)